pkg/database/vercode.go

```diff
@@ -97,11 +98,22 @@ func (v *VerificationCode) AfterCreate(scope *gorm.Scope) {
 				SET codes_issued = authorized_app_stats.codes_issued + 1
 		`
 
-		day := time.Now().UTC().Truncate(24 * time.Hour)
 		if err := scope.DB().Exec(sql, day, v.IssuingAppID).Error; err != nil {
 			scope.Log(fmt.Sprintf("failed to update stats: %v", err))
 		}
 	}
+
+	// Update the per-realm stats.
+	sql := `
```

Comment:
Yea, if you move this into a callback (see the other stats calculations), we can avoid the duplication.

Reply:
Done.

Reply:
Did you push? I'm not seeing any changes.